Cargo.toml

```diff
@@ -46,6 +46,7 @@ picoserve = { version = "0.12.2", features = ["embassy", "alloc"] }
 embassy-sync = "0.6.0"
 wot-td = { version = "0.6.2", default-features = false, features = ["alloc"] }
 serde_json = { version = "1.0.133", default-features = false, features = ["alloc"] }
+nuuid = { version = "0.5.0", default-features = false }
```

Comment:
uuid seems more maintained and is also no_std, I'd switch to it.

Reply:
Thank you :) I now made the switch to the `uuid` crate and it still works, apparently.